rllib/offline/offline_data.py

```diff
@@ -223,7 +223,7 @@ def sample(
     @property
     def default_read_method_kwargs(self):
         return {
-            "override_num_blocks": max(self.config.num_learners * 2, 2),
+            # "override_num_blocks": max(self.config.num_learners * 2, 2),
```

Comment:
Is this line not needed?

Reply:
Great catch. Somehow this makes it into multiple of my PRs.